conf/dokuwiki.php

```diff
@@ -50,6 +50,7 @@
 $conf['useheading']  = 0;                 //use the first heading in a page as its name
 $conf['sneaky_index']= 0;                 //check for namespace read permission in index view (0|1) (1 might cause unexpected behavior)
 $conf['hidepages']   = '';                //Regexp for pages to be skipped from RSS, Search and Recent Changes
+$conf['ack_page_nearest'] = 0;
```

Comment:
Please add a brief inline comment as other config options.

Reply:
Thanks. Missed that. Fixed in latest commit.